frameworks/cassandra/tests/config.py

```diff
@@ -208,6 +201,44 @@ def get_write_data_job(
     )
 
 
+def get_write_udf_job(
+    node_address: str = DEFAULT_NODE_ADDRESS,
+    node_port: str = DEFAULT_NODE_PORT,
+    dcos_ca_bundle: Optional[str] = None,
+) -> Dict[str, Any]:
+    cql = " ".join(
+        [
+            "CREATE KEYSPACE testspace1 WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 3 };",
+            "USE testspace1;",
+            "CREATE TABLE testtable1 (id int, val1 int, val2 int, PRIMARY KEY(id));",
+            "INSERT INTO testspace1.testtable1(id, val1, val2) VALUES(1, 100, 200);",
+            "CREATE OR REPLACE FUNCTION maxof(currentvalue int, testvalue int) RETURNS NULL ON NULL INPUT RETURNS int LANGUAGE java AS 'return Math.max(currentvalue,testvalue);';",
+        ]
+    )
+    return _get_test_job(
+        "write-data",
+        [_cqlsh(cql, node_address, node_port)],
+        node_address,
+        node_port,
+        dcos_ca_bundle=dcos_ca_bundle,
+    )
+
+
+def get_verify_udf_data_job(
+    node_address: str = DEFAULT_NODE_ADDRESS,
+    node_port: str = DEFAULT_NODE_PORT,
+    dcos_ca_bundle: Optional[str] = None,
+) -> Dict[str, Any]:
+    cmds = [
+        "{} | grep 200".format(
+            _cqlsh("SELECT maxOf(val1,val2) FROM testspace1.testtable1;", node_address, node_port)
```

Comment:
ha, looks like the function names are case-insensitive since you created the function as `maxof()`

Reply:
ye abut updated it as well.